### PR TITLE
New Metrics Template Update:  Issue-label-inclusivity.md

### PR DESCRIPTION
**Pull request by peculiaruc**
Updated the Organizational Influence using the new metrics template

- Tweak the description and the objective sections to form the Overview section.
- Remove Tool Providing metric section e.t.c

### CODE REVIEW

File: focus-areas/project-and-community/issue-label-inclusivity.md [hunk 1]
```diff
@@ -1,25 +1,33 @@
...
 
+**Question: How well are project issues labelled to invite new contributors, skilled contributors, non-code contributors, and other types of contributors?**
 
-## Description
-Issue label inclusivity helps gauge the affordability of issues to different contributors at different levels of familiarity (e.g., newcomer, occasional contributor, and core contributors), with different skills (e.g., languages, frameworks, APIs, documentation, frontend, and backend), and with different goals (e.g., code and non-code contributions).
 
-## Objectives
+## Overview
+Issue Label Inclusivity measures the accessibility and inclusiveness of project issues for contributors of various skill levels and backgrounds. Data points include the number and types of labels used, the clarity of issue titles and descriptions, and the presence of labels that indicate the level of skills needed to tackle the issue. The Issue Label Inclusivity metric can help a project ensure: newcomer friendliness issues are created, mentor availability, issue list diversity, usable titles and descriptions and consistent use of labels to identify types of contributor or skill level needed to work on an issue.
```
Comment (germonprez) [inline focus-areas/project-and-community/issue-label-inclusivity.md:7]: Recommended edits: 

Issue Label Inclusivity measures the accessibility and inclusiveness of project issues for contributors of various skill levels and backgrounds. Data points include the number and types of labels used, the clarity of issue titles and descriptions, and the presence of labels that indicate the level of skills needed to tackle the issue. The Issue Label Inclusivity metric can help a project ensure: newcomer friendly issues are created, issue list diversity, usable titles and descriptions, and consistent use of labels to identify types of contributor or skill level needed to work on an issue.

File: focus-areas/project-and-community/issue-label-inclusivity.md [hunk 2]
```diff
@@ -1,25 +1,33 @@
...
+Inclusive labelling practices help to create a more equitable and accessible environment for contributors from all backgrounds. Providing clear and informative labels, helps  projects reduce the learning curve for new contributors, encourage participation and promote diversity within the community.
```
Comment (ElizabethN) [inline focus-areas/project-and-community/issue-label-inclusivity.md:8]: Suggested edits:

Inclusive labelling practices help create a more equitable and accessible environment for contributors from all backgrounds. Providing clear and informative labels helps projects reduce the learning curve for new contributors, encourages participation, and promotes diversity within the community.

File: focus-areas/project-and-community/issue-label-inclusivity.md [hunk 3]
```diff
@@ -1,25 +1,33 @@
...
+   - *Inclusive labels*:
+     - Newcomer friendly ones look for (*newcomer*, *good first issue*) in the project’s list of labels and in the issue list (labels in use)
+     - Mentor friendly ones look for (*mentor*) in the project’s list of labels and in the issue list (labels in use)
+
+  - *Skill labels*:
+     - Presence of labels to identify skills needed (e.g, Java, Python, HTML, machine learning) in the project’s list of labels and in the issue list (labels in use)
+
+- Observe the frequency of each label used across issues in a project
+
+
```
Comment (germonprez) [inline focus-areas/project-and-community/issue-label-inclusivity.md:30]: Could you please remove the '(labels in use)' from the end of every sentence?

File: focus-areas/project-and-community/issue-label-inclusivity.md [hunk 4]
```diff
@@ -1,25 +1,33 @@
...
+      - Presence of labels to identify general needs of  “Feature”, “Bug”, and “Documentation” label”, a “Front End”, and a “Back End” label in the project’s list of labels and in the issue list (labels in use)
+   - *Inclusive labels*:
+     - Newcomer friendly ones look for (*newcomer*, *good first issue*) in the project’s list of labels and in the issue list (labels in use)
+     - Mentor friendly ones look for (*mentor*) in the project’s list of labels and in the issue list (labels in use)
+
+  - *Skill labels*:
+     - Presence of labels to identify skills needed (e.g, Java, Python, HTML, machine learning) in the project’s list of labels and in the issue list (labels in use)
+
+- Observe the frequency of each label used across issues in a project
+
```
Comment (ElizabethN) [inline focus-areas/project-and-community/issue-label-inclusivity.md:29]: Suggested edit:

- Identify the published list of issue labels used for each project
   - *General labels* identify general needs of the project (e.g., *Feature*, *Bug*, and *Documentation*)
   - *Newcomer-friendly labels* identify issues that are appropriate for first-time contributors (e.g., *newcomer*, *good first issue*)
   - *Skill labels* identify skills needed (e.g, *Java*, *Python*, *HTML*, *machine learning*)

- Observe the frequency of each label used across issues in a project

File: focus-areas/project-and-community/issue-label-inclusivity.md [hunk 5]
```diff
@@ -40,24 +48,26 @@ The Issue Label Inclusivity metric can help a project ensure:
 
 From: https://github.com/kubernetes/kubernetes/labels
```
Comment (ElizabethN) [inline focus-areas/project-and-community/issue-label-inclusivity.md:49]: Can you please remove this line?

File: focus-areas/project-and-community/issue-label-inclusivity.md [hunk 6]
```diff
@@ -1,25 +1,33 @@
...
+
+  - *Skill labels*:
+     - Presence of labels to identify skills needed (e.g, Java, Python, HTML, machine learning) in the project’s list of labels and in the issue list (labels in use)
+
+- Observe the frequency of each label used across issues in a project
+
+  
 ### Filters
 
 - Type of label
```
Comment (ElizabethN) [inline focus-areas/project-and-community/issue-label-inclusivity.md:33]: Can you change this to "none identified."

**Review by ElizabethN — CHANGES_REQUESTED** [review]

**Comment by peculiaruc** [comment]
All comments resolved

**Review by ElizabethN — APPROVED** [review]

**Pull request merged**